apisix/plugins/limit-count/limit-count-redis.lua

```diff
@@ -85,27 +70,52 @@ function _M.incoming(self, key)
         -- core.log.info(" err: ", err)
         return nil, err
     end
+    return red, nil
+end
+
+function _M.new(plugin_name, limit, window, conf)
+    assert(limit > 0 and window > 0)
+
+    local self = {
+        limit = limit,
+        window = window,
+        conf = conf,
+        plugin_name = plugin_name,
+    }
+    return setmetatable(self, mt)
+end
+
+function _M.incoming(self, key)
+    local conf = self.conf
+    local red, err = redis_cli(conf)
+    if err then
+        return red, err
```

Comment:
```suggestion
        return nil, err, 0
```
Please follow the code style: https://github.com/apache/apisix/blob/master/CODE_STYLE.md#error-handling

Reply:
fixed